include/avif/avif.h

```diff
@@ -665,9 +665,11 @@ typedef struct avifImage
 // avifImageCreate() and avifImageCreateEmpty() return NULL if arguments are invalid or if a memory allocation failed.
 AVIF_API avifImage * avifImageCreate(uint32_t width, uint32_t height, uint32_t depth, avifPixelFormat yuvFormat);
 AVIF_API avifImage * avifImageCreateEmpty(void); // helper for making an image to decode into
-// Performs a deep copy of an image, including all metadata and planes, and the gain map metadata/planes if present.
+// Performs a deep copy of an image, including all metadata and planes, and the gain map metadata/planes if present
+// and AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP is defined.
```

Comment:
Nit/optional: The following part of this sentence could be edited:

> and the gain map metadata/planes if present and AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP is defined

I don't have a good suggestion. I think we should at least add an "if":

> and the gain map metadata/planes if present and if AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP is defined

Sorry about the nonconstructive criticism.